README.md

```diff
@@ -599,7 +599,15 @@ The following example was "contrived". The `drupal_loadtest` example was run for
  Aggregated               |      432.98 |     294.11 |       3,390 |         14
  ```
 
-From these two tables, it is clear that there was a statistically significant event affecting the load testing metrics. In particular, note that the standard deviation between the "raw" average and the "adjusted" average is considerably larger than the "raw" average, calling into questing whether or not your load test was "valid". (The answer to that question depends very much on your specific goals and load test.)
+Note: It is beyond the scope of Goose to test for statistically significant changes in the right-tail, or other locations, of the distribution of response times.  Goose produces the raw data you need to conduct these tests.
+
+Nonetheless, for users interested in establishing if there was an event(s) affecting the shape of the distribution of load test metrics (by a statistically significant amount): The following program is a reasonable starting point.
+
+1. Run a test without the CO event(s), record the raw response data.
+2. Run a test with the CO events, record the CO-adjusted response data.
```

Comment:
It's generally not possible to run the same load test twice, simulating the CO event one of the times, in my general use-case for Goose. For creating CO Mitigation I of course did exactly this, but in "real world" tests this isn't something I'd be doing.

Goose is in fact detecting a probably CO event (ie, when the cadence from on or more `GooseUser` threads takes greater than 2x as long as is normal) and reporting on this. Further, it also back-fills the metrics (as HdrHistogram does) with what it statistically would have expected to see if all other requests by the GooseUser had been made during the event.

Reply:
> It's generally not possible to run the same load test twice, simulating the CO event one of the times...

Correct.  But it is possible to run the same load test twice.  I see now the ambiguity.  Hope this clarifies:

1. Run a test in circumstances where you believe the test can serve as a baseline sample from the 'healthy' state, record the raw response data.
2. Run a test in circumstances where you want to compare the distribution of response times against your baseline, record the CO-adjusted response data.

Better?